packages/itwin/measure-tools/src/measurements/AreaMeasurement.ts

```diff
@@ -36,12 +36,17 @@ import { MeasurementSelectionSet } from "../api/MeasurementSelectionSet.js";
 import { Polygon } from "../api/Polygon.js";
 import { DistanceMeasurement } from "./DistanceMeasurement.js";
 import { MeasureTools } from "../MeasureTools.js";
+import type { FormatterSpec } from "@itwin/core-quantity";
 
 /**
  * Props for serializing a [[AreaMeasurement]].
  */
 export interface AreaMeasurementProps extends MeasurementProps {
   polygonPoints: XYZProps[];
+  lengthKoQ?: string;
```

Comment:
@grigasp should I wrap these new properties inside of one object? Like
```ts
...
formattingSpecProps: {
  lengthKoQ?: string;
  lengthPersistenceUnitName?: string;
  areaKoQ?: string;
  areaPersistenceUnitName?: string;
}
```

Could declare the above as it's own `AreaFormattingSpecProps`, and shorten the static create method at the bottom of this file... Fyi, that create method is the main entry point for the higher level APIs to instantiate these Measurement subclasses (up to the app level and them registering a MeasureToolsUiItemsProvider)

Reply:
I'm not familiar with APIs in this package, so don't have a strong opinion... But I like the idea of grouping similar props together. In fact, I'd probably go a step further and do this:
```ts
formatting: {
  length?: {
    /** Defaults to `AecUnits.LENGTH` */
    koqName?: string;
    /** Defaults to `Units.M` */
    persistenceUnitName?: string;
  },
  area?: {
    /** Defaults to `AecUnits.AREA` */
    koqName?: string;
    /** Defaults to `Units.SQ_M` */
    persistenceUnitName?: string;
  },
}
```